### ISSUE TITLE
Untranslated string in SI reply interface

### ISSUE THREAD

**Issue by eaon** [freedomofpress/securedrop#6279]
Just stumbled over [a visually hidden header](https://github.com/freedomofpress/securedrop/blob/4a80f70901f49dfe2f30b79d2a9bbc69cf807cec/securedrop/source_templates/lookup.html#L83) in the reply section that is missing from translations. Because it's hidden it only affects screen readers.

A quick look at the `message.pot` history makes me believe this is a new string that we missed during review.

**Comment by ninavizz** [comment]
A note to my successor: usability testing for vision-impaired folk, pls!! <3

**Comment by cfm** [comment]
Thanks for catching and tracing this, @eaon.  The code-level fix is trivial but, as you note, will add a new source string for translation.  Unless we think we can make that happen today, I think, reluctantly, that we'll have to defer it to v2.2.1.

(NB.  Since freedomofpress/securedrop-client#1272, the Client's gettext coverage is checked and enforced by CI.  I'll file a follow-up ticket for whether there's a reasonable way to implement the same for the Server, given the greater complexity of its HTML/Jinja templates.)

**Comment by legoktm** [comment]
>  Unless we think we can make that happen today, I think, reluctantly, that we'll have to defer it to v2.2.1.

I doubt we will get any translations in the short time window so even if we did this, everyone would get the English string anyways, so deferring to 2.2.1 makes sense to me too.

**Comment by eaon** [comment]
I think we should try and get this translated before the string freeze. In my SI refactor branch I opted for this:

`{{ gettext('Delete reply from {relative_time}?').format(relative_time=relatime) }}`

where `{relative_time}` is filled using a macro from [utils.html](https://github.com/eaon/securedrop/blob/39da05e4cb68ace1e80992bea0ee110533823376/securedrop/source_templates/utils.html) to create a time tag. Is that a reasonable approach @cfm? Or is there a best practice I'm not familiar with?

**Comment by cfm** [comment]
That looks good to me, @eaon.  The `{relative_time}` placeholder will be preserved in the source string shown to translators and required to be present in their translations.  Out of context in Weblate, something like `{timestamp}` might be clearer to translators, but that's a nit.
